web-ng/src/app/components/map/map.component.ts

```diff
@@ -179,6 +180,18 @@ export class MapComponent implements AfterViewInit, OnDestroy {
     });
   }
 
+  private panAndZoom(
+    position?: google.maps.LatLng | google.maps.LatLngLiteral
```

Comment:
I assume you did this because this is the type defined in MapOptions? For this method, can we instead accept required google.maps.LatLng?